system/Session/Handlers/BaseHandler.php

```diff
@@ -50,7 +50,7 @@
 	/**
 	 * The Data fingerprint.
 	 *
-	 * @var boolean
+	 * @var bool
```

Comment:
Are you using the phpcbf pre-commit hook? It changes bool to boolean

Reply:
Hi, no I'm not... Should I? In other words, I'm trying to use bool as a primitive type instead of boolean as eventually a class/interface name. Is this correct? I'm talking about this:
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

Reply:
The phpcbf pre-commit hook will "fix" style things. Specifically, if you change "boolean" to "bool", it should change that back to "boolean" before letting you proceed. The codeigniter4/standards repo rule set needs to be fixed to resolve this.
And yes, you should be using the pre-commit hook. That is in our composer.json, and will be installed automatically if you do a "composer update" in your fork's clone.

Reply:
ps - your reasoning is good, but phpcbf will automatically "undo" your changes.